sdk/peering/Microsoft.Azure.Management.Peering/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Peering Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Peering management functions for managing the Microsoft Azure Peering service.")]
 
-[assembly: AssemblyVersion("0.12.0.0")]
-[assembly: AssemblyFileVersion("0.12.0.0")]
+[assembly: AssemblyVersion("0.13.0.0")]
```

Comment:
Use the same version as in csproj
```suggestion
[assembly: AssemblyVersion("2.0.0.0")]
```